src/models.rs

```diff
@@ -72,3 +72,23 @@ pub struct NewMetric {
     pub metric_name: String,
     pub data: serde_json::Value,
 }
+
+#[derive(Serialize, Debug)]
+pub struct ErrorObject {
+    pub message: String,
+    // pub code: usize,
+}
+
+#[derive(Serialize, Debug)]
+pub struct Error {
+    pub errors: Vec<ErrorObject>,
+}
+
+#[derive(Responder, Debug)]
+pub enum ErrorResponder {
+    #[response(status = 400, content_type = "json")]
+    UserError(Json<Error>),
```

Comment:
Maybe call this `BadRequest` instead? Makes sense to me to use the standard name